Prepare release v47

aclemons opened the pull request.

aclemons committed:
Bump version number

rg3 commented:
Minor additional change request: replace http://rg3.github.io/slackroll/ with https://rg3.github.io/slackroll/ in slackroll.slackbuild, manpage.txt, helpfiles/00-default.txt and helpfiles/all.txt, then rebuild the package. I saw it while reviewing the package now, sorry.

aclemons committed:
Update all urls to https

aclemons commented:
No problem. Pushed up the changes now.

aclemons committed:
Point to the frequently asked questions in the man page

There is no longer a separate page, so mentioning a link to the
frequently asked questions document is incorrect.
